spot-gen/README.txt

```diff
@@ -0,0 +1,5 @@
+Simple data generation
```

Comment:
Done. This data generator allows to produce logs (simulate some activity). It can be generated, loaded and parsed on regular basis so any reports like "show some information for last hour/day/week" will be actual and non-empty.
It can be used for testing or demonstrating purposes